website/content/docs/secrets/azure.mdx

```diff
@@ -300,6 +300,18 @@ Azure service principal configured with the same permissions.
   `ffffff`. These may be used to search for Vault-created credentials using the `az` tool
   or Portal.
 
+## Azure debug logs
+
+The Azure secret engine plugin supports debug logging which includes additional information
+about requests and responses from the Azure API.
+
+To enable the Azure debug logs, set the following environment variable on the Vault
```

Comment:
```suggestion
To enable the Azure debug logs, set the `AZURE_GO_SDK_LOG_LEVEL`  environment variable to `DEBUG` on your Vault
```

Style correction: rather than using spatial orientation to refer to the relevant environment variable, you can just name the variable and the value folks should use, then provide the example code to demonstrate how to actually set the variable correctly.